action.yml

```diff
@@ -17,9 +17,9 @@ inputs:
     description: 'Gradle wrapper run command'
     required: false
     default: './gradlew --no-daemon --info clean build'
-  sonar-token:
-    description: 'Token used for triggering and uploading the static code analysis to Sonar Cloud'
-    required: false
+  sonar:
```

Comment:
Seems sonar-token wasn't used to scan the projects, our sonarclod is auto-configured with GitHub orgs.

Though this change will require us to update all our pipelines for ecs-build-gradle, but we can do it whenever we work on a specific repo which uses this action.